doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -403,60 +417,209 @@ Response code:
             }
         }
 
-### Retrieve entity data by Id. Not exists [GET /v2/entities/{entityId}]
+### Update or append entity attributes [POST /v2/entities/{entityId}?{op}]
+
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">op</td>
+  <td>Operation type. Allowed values: <code>append</code>.</td>
+</tr>
+</table>
+
+The request payload is an object representing the attributes to append or update. The object follows
```

Comment:
Not sure if a good way of describing the kind of payload this operation, I mean to refer to the JSON entity representation then ammend with "except that `id` and `type` are not allowed". Maybe it is a bit dirty, but it saves adding a whole new section for "JSON attributes list representation" which basically would have the same text than the section in "JSON entity representation format".

What do you think?